deb/after_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I think it is better to use `/bin/sh` (Bourne Shell) rather than bash for deb package after install script.